src/content/configuration/dev-server.md

```diff
@@ -20,12 +20,15 @@ contributors:
   - g100g
   - anikethsaha
   - snitin315
+  - maxloh
 ---
 
 [webpack-dev-server](https://github.com/webpack/webpack-dev-server) can be used to quickly develop an application. See the [development guide](/guides/development/) to get started.
 
 This page describes the options that affect the behavior of webpack-dev-server (short: dev-server).
 
+T> dev-server uses `output.path` instead of current working directory to be the root path for serving generated files, which might not work on your project structure (e.g. if you reference output files with relative path). See the [development guide](/guides/development/#using-webpack-dev-server) for more information on adjusting webpack configuration to work with dev-server.
+
```

Comment:
A tip in the API documentation page will be more easy to spot as people tends to find solutions in API documentations instead of guides.

I can remove it though if you really want.

Reply:
There're already a lot of tips/warnings,

![image](https://user-images.githubusercontent.com/1091472/107115884-8df05900-68aa-11eb-9948-c12b1a120371.png)

It's so distracting in my opinion. We can't cover every basic concepts on API documentation, guides serve that purpose well.

Reply:
OK